src/components/learner-credit-management/BudgetDetailPageOverviewAvailability.jsx

```diff
@@ -101,10 +101,8 @@ const BudgetActions = ({ budgetId, isAssignable, enterpriseId }) => {
           className="mt-3"
           iconBefore={Add}
           as={Link}
-          to={{
-            pathname: generatePath(routeMatch.path, { budgetId, activeTabKey: 'catalog' }),
-            state: { budgetActivityScrollToKey: 'catalog' },
-          }}
+          to={generatePath(pathname, { budgetId, activeTabKey: 'catalog' })}
```

Comment:
Noticed a bug with this where when the user is on the `activity` tab and selects the New course assignment button, the tab no longer changes to the catalog tab.  The same behavior can be seen from clicking the `View spent activity` and `View assigned activity` button from the utilization details button (See Video).  For those buttons, when the user is on the catalog tab, the tab is supposed to switch to the activity tab and scroll down to the selected component.  (Found in `BudgetDetailPageOverviewUtilization`

https://github.com/openedx/frontend-app-admin-portal/assets/82611798/dc78bfc1-8d69-4312-a8e3-b164936e5fd1